trade/trader_stakecube.js

```diff
@@ -127,20 +129,22 @@ module.exports = (
     async getBalances(nonzero = true) {
       const paramString = `nonzero: ${nonzero}`;
 
-      let userData;
+      let scData;
 
       try {
-        userData = await stakeCubeApiClient.getUserData();
+        scData = await stakeCubeApiClient.getUserData();
 
-        if (userData.errorMessage) {
-          log.warn(`API request getBalances(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${userData.errorMessage}`);
+        if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.